Oracle/diagnosis.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE diagnosis(
 	ENCOUNTERID varchar(50) NOT NULL,
 	ENC_TYPE varchar(2) NULL,
 	ADMIT_DATE date NULL,
+    DX_DATE date NULL,
```

Comment:
any particular reason to indent this differently?

Do the style guidelines in CONTRIBUTING.md need updating?

Reply:
changed the indent. for some reason in sql developer tab was getting converted to 4 spaces. not sure why.